docs/application/native/guides/text-input/tts-engine.md

```diff
@@ -0,0 +1,369 @@
+# Text-to-speech Engine
+
+
+Text-to-speech Engine (TTSE) is an engine to synthesize voice from input texts and to send the synthesized sound data to TTS framework. 
+
+TTSE developers can provide TTSE service users, to apply TTSE with functions that is necessary to operate the engine.
+
+The main features of the TTSE API include:
+
+- Preparing the TTSE service for use
+
+  You can connect the TTSE service application. [Basic TTSE Processes](#basic) and [Prerequisites](#prerequisites) are defined.
+
+- TTSE Parameters
+
+  You can [set information](#parameters), which includes language and state.
+
+- TTSE Retrieval information
+
+  You can [get information](#info_retrieval), which includes command count and audio type.
+
+- TTSE Send results
+
+  You can send the results to the engine service user. [TTSE Send results](#send_result), is defined.
```

Comment:
```suggestion
  You can send the results to the engine service user. [TTSE send results](#send_result), is defined.
```

Reply:
I changed this part like the followings:
```suggestion
- Send results

  You can [send the results](#send_result) to the engine service user.
```

Please review this one more time.